packages/cells/src/widget.ts

```diff
@@ -572,7 +583,12 @@ class CodeCell extends Cell {
     });
 
     // Modify state
+    const metadataScrolled = model.metadata.get('scrolled') as boolean | 'auto' | undefined;
+    this.outputsScrolled = metadataScrolled === 'auto' ? false : metadataScrolled || false;
```

Comment:
Same here. I think we can be a bit more defensive about the values coming out of the settings (I am speaking from experience of cryptic failures due to unexpected values in the settings JSON).

Reply:
Sounds good.